lib/oprtglat.gi

```diff
@@ -463,8 +463,8 @@ local n,l, o, b, t, r;
   # new code -- without `all` option
 
   n:=Length(dom);
-  if n>20 and ForAll(dom,x->IsSubset(G,x))
-    and NrMovedPoints(G)>1000 then
+  sub:=ForAll(dom,x->IsSubset(G,x));
+  if n>20 and sub and NrMovedPoints(G)>1000 then
     #and NrMovedPoints(G)*1000>Size(G) then
```

Comment:
Could also just TryNextMethod here and get rid of `sub` again *shrug*
```suggestion
  if not ForAll(dom,x->IsSubset(G,x)) then TryNextMethod(); fi;
  if n>20 and NrMovedPoints(G)>1000 then
```

Reply:
Yes. This is a fix, and I thus wanted to make the structure as simple and clear as possible. Ultimately I want to be able to get rid of this subset condition again.